gluten-delta/src/test/scala/io/glutenproject/execution/VeloxDeltaSuite.scala

```diff
@@ -183,4 +183,21 @@ class VeloxDeltaSuite extends WholeStageTransformerSuite {
       )
     }
   }
+
+  testWithSpecifiedSparkVersion("deletion vector", Some("3.4.2")) {
```

Comment:
Why limits to `3.4.2`?

Reply:
This feature is only availabel after delta 2.3. for spark 3.2 and 3.3 we use delta 2.0 and 2.2.
https://docs.delta.io/latest/delta-deletion-vectors.html